src/tests/CMakeLists.txt

```
@@ -14,16 +13,19 @@ set(sources
     TimerTests.cpp
     StreamTestUtils.cpp
     RangeSetTests.cpp
-    $<TARGET_OBJECTS:__objects>
     ListenerTests.cpp
     BrokerSettingsTests.cpp
     ConfStandIn.h
     ProducerDeliveryCbTests.cpp
     ConsumerTests.cpp
     MockMessage.h
     UnitTests.cpp)
-add_executable(${tgt} ${sources})
-add_dependencies(${tgt} flatbuffers_generate)
-target_include_directories(${tgt} PRIVATE ${path_include_common})
-target_link_libraries(${tgt} ${libraries_common})
-add_gtest_to_target(${tgt})
+add_executable(tests ${sources})
```

I really like that you have removed this `${tgt}`-nonsense. With that said, I do prefer to use a (de-facto standard) `*_SRC` name.
```suggestion
add_executable(tests ${tests_SRC})
```